ArduCopter/crash_check.cpp

```diff
@@ -1,7 +1,13 @@
 #include "Copter.h"
 
 // Code to detect a crash main ArduCopter code
-#define CRASH_CHECK_TRIGGER_SEC         2       // 2 seconds inverted indicates a crash
+#if PARACHUTE == ENABLED
+    // CRASH_CHECK_TRIGGER_SEC should be grater than CHUTE_TIMEOUT, to prevent disarming without deploying parachute
+    #define CRASH_CHECK_TRIGGER_SEC parachute.get_chute_timeout() + 1.0f
```

Comment:
this define is not doing what you want, as it is used in a multiplication, you need parantheses around the define 
```
    #define CRASH_CHECK_TRIGGER_SEC  (parachute.get_chute_timeout() + 1.0)
```